nanorc.nanorc

```diff
@@ -3,47 +3,71 @@
+## Possible invalid parameters
+color brightred "^[[:space:]]*((un)?set|include|syntax|i?color).*$"
+
+## Basic colors
+color red       "\<red\>"
+color green     "\<green\>"
+color blue      "\<blue\>"
+color magenta   "\<magenta\>"
+color yellow    "\<yellow\>"
+color cyan      "\<cyan\>"
+color white     "\<white\>"
+color black     "\<black\>"
+color normal    "\<normal\>"
+## Light/bright colors
+color lightred          "\<(light|bright)red\>"
+color lightgreen        "\<(light|bright)green\>"
+color lightblue         "\<(light|bright)blue\>"
+color lightmagenta      "\<(light|bright)magenta\>"
+color lightyellow       "\<(light|bright)yellow\>"
+color lightcyan         "\<(light|bright)cyan\>"
+color lightwhite        "\<(light|bright)white\>"
+color lightblack        "\<(light|bright)black\>"
+color normal            "\<(light|bright)normal\>"
+## 256 colors
+color pink      "\<pink\>"
+color purple    "\<purple\>"
+color mauve     "\<mauve\>"
+color lagoon    "\<lagoon\>"
+color mint      "\<mint\>"
+color lime      "\<lime\>"
+color peach     "\<peach\>"
+color orange    "\<orange\>"
+color latte     "\<latte\>"
+color rosy      "\<rosy\>"
+color beet      "\<beet\>"
+color plum      "\<plum\>"
+color sea       "\<sea\>"
+color sky       "\<sky\>"
+color slate     "\<slate\>"
+color teal      "\<teal\>"
+color sage      "\<sage\>"
+color brown     "\<brown\>"
+color ocher     "\<ocher\>"
+color sand      "\<sand\>"
+color tawny     "\<tawny\>"
+color brick     "\<brick\>"
+color crimson   "\<crimson\>"
 
 ## Keywords
-icolor brightgreen "^[[:space:]]*(set|unset)[[:space:]]+(afterends|allow_insecure_backup|atblanks|autoindent|backup|backupdir|boldtext|brackets|breaklonglines|casesensitive|constantshow|cutfromcursor|emptyline|errorcolor|fill|functioncolor|guidestripe|historylog|jumpyscrolling|keycolor|linenumbers|locking|matchbrackets|morespace|mouse|multibuffer|noconvert|nohelp|nonewlines|nopauses|nowrap|numbercolor|operatingdir|positionlog|preserve|punct|quickblank|quotestr|rawsequences|rebinddelete|regexp|selectedcolor|showcursor|smarthome|smooth|softwrap|speller|statuscolor|stripecolor|suspend|tabsize|tabstospaces|tempfile|titlecolor|trimblanks|unix|view|whitespace|wordbounds|wordchars|zap)\>"
-icolor green "^[[:space:]]*(bind|set|unset|syntax|header|include|magic)\>"
+color green "^[[:space:]]*(set|unset|syntax|header|magic|formatter|linter|comment|tabgives|include|extendssyntax|bind|unbind)\>"
+color white "\<(bold|italic)\>" ","
+color magenta  "^[[:space:]]*i?color\>" "\<(start|end)="
+#color yellow   "^[[:space:]]*(set|unset)[[:space:]]+(errorcolor|functioncolor|keycolor|numbercolor|selectedcolor|statuscolor|stripecolor|titlecolor)[[:space:]]+(bright|light)?(white|black|red|blue|green|yellow|magenta|cyan|normal)?(,(white|black|red|blue|green|yellow|magenta|cyan|normal))?\>"
```

Comment:
I removed this part because otherwise we will have to enumerate all possible colors here. Instead, I think let them be colored as their own color would be good enough.